tutorials/demo_dataset/make.py

```diff
@@ -0,0 +1,29 @@
+import uuid
```

Comment:
Let's remove this module and update the `get_demo_data` from `datatracer/data.py` instead.

Also remove the CSVs from inside the tutorials folder and call the `get_demo_data` function in the tutorial, so users can run `pip install datatrfcer` and directly run the tutorial from any environment without the need to having cloned the repository first.

Reply:
Let's circle back on this, we're thinking about replacing this demo dataset entirely with something like https://relational.fit.cvut.cz/dataset/SalesDB.